HW_3/forms.py

```diff
@@ -0,0 +1,10 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, IntegerField, SelectField, PasswordField, SubmitField
+from wtforms.validators import DataRequired, EqualTo, Email, Length
+
+class RegistrationForm(FlaskForm):
+    name = StringField('Username', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Почему не `EmailField`?
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```